.changeset/yellow-lions-double.md

```diff
@@ -0,0 +1,5 @@
+---
+'@openfn/language-commcare': patch
+---
+
+Update commcare request docs to read query parameters instead of options
```

Comment:
This needs updating. The behavioural change is way more important than the docs change

Reply:
@josephjclark I have fixed it